CONTRIBUTING.md

```diff
@@ -56,6 +58,13 @@ This project follows [Google's Open Source Community Guidelines](https://opensou
 - Use descriptive names that reflect the purpose of the branch.
 - Add `issue tracker id`
 
+
+**Example:** `feat/<issue-id>/gcp-compute-engine-support`
```

Comment:
You need to have new line in order to have it as a list and not like a sentence. Additionally, you have duplicate feat/issue-id now. PTAL at the final result.
https://github.com/ro4i7/gcp_scanner/blob/patch1/CONTRIBUTING.md